platforms/ios/framework/src/TGMapView.mm

```diff
@@ -369,6 +381,8 @@ - (void)glkView:(GLKView *)view drawInRect:(CGRect)rect
         return;
     }
 
+    [self validateContext];
```

Comment:
This check is unnecessary, I believe. According to the GLKView docs the context is set before this delegate method is called: https://developer.apple.com/documentation/glkit/glkviewdelegate/1615595-glkview?language=objc

Reply:
Yup, that documentation is good. The fix is still holding its ground.